source/fuzz/fuzzer_util.h

```diff
@@ -64,6 +64,12 @@ void AddUnreachableEdgeAndUpdateOpPhis(
     bool condition_value,
     const google::protobuf::RepeatedField<google::protobuf::uint32>& phi_ids);
 
+// Returns true if and only if |loop_header_id| is in fact loop header and
```

Comment:
"is in fact loop header" -> "is a loop header"

Reply:
Done.